monitoring/uss_qualifier/scenarios/astm/netrid/display_data_evaluator.py

```diff
@@ -95,6 +99,14 @@ class TelemetryMapping(object):
     observed_flight: ObservationType
 
 
+@dataclass
+class FetchedToInjectedCache(object):
+    # Mapping is a map of URL -> ParticipantID that we found until now
+    mappings: dict[str, ParticipantID] = field(default_factory=dict)
+    # Unmapped is the list of flights we didn't attributed yet
+    unmapped: list[FetchedUSSFlights] = field(default_factory=list)
```

Comment:
I don't want to hold up this PR for it since this is not intended as a public class, but docstrings should follow PEP257 -- specifically, the docstring should be a string literal **following** the thing being documented and, by convention, should use triple quotes.

```suggestion
    mappings: dict[str, ParticipantID] = field(default_factory=dict)
    """A map of URL -> ParticipantID that we found until now"""
    
    unmapped: list[FetchedUSSFlights] = field(default_factory=list)
    """List of flights we didn't attributed yet"""
```